bundles/org.openhab.binding.miio/README.md

```diff
@@ -1367,7 +1367,7 @@ e.g. `smarthome:send actionCommand 'upd_timer["1498595904821", "on"]'` would ena
 
 | Channel          | Type    | Description                         |
 |------------------|---------|-------------------------------------|
-| vacuumaction     | String  | Vacuum Action                       |
+| dreameControl    | String  | Vacuum Action                       |
```

Comment:
Renaming the channel breaks backward compatibility, but the description remains the same. Is it needed to rename therefor?

Reply:
The mean reason for the change is that the type is changing from number to a string... meaning that it will most likely break backwards anyway.
As this type was added very recent (think it is not in 2.5.8) little chance that many users already have this.

nevertheless, I'll change the name back to the earlier channel name and only change the channelType

Reply:
If it was not in 2.5.8 it's not officially released yet so we don't consider it a breaking change than.